webapp-django/crashstats/signature/views.py

```diff
@@ -195,7 +193,7 @@ def signature_reports(request, params):
 
     context['current_url'] = '%s?%s' % (
         reverse('signature:signature_report'),
-        current_query.urlencode()
+        current_query.urlencode().replace('+', '%20')
```

Comment:
Perhaps consider doing `urlencode_dict(current_query)` and inside `urlencode_dict` you do a 
```python
if hasattr(thing, 'urlencode'):
    thing = urllib.urlencode()
else:
    thing = urllib.urlencode(thing, True)
# comment here WHY we have to do this
return thing.replace('+', '%20')
```
that way, the hacky `+` replace thing is only in one place.